proto/swapi.proto

```diff
@@ -77,6 +78,8 @@ service Starwars {
        get: "/swapi/v1/people"
     };
   }
+
+  rpc ListStarshipActions(ListStarshipActionsRequest) returns (stream StarshipAction) {}
```

Comment:
This is server streaming no?

Reply:
> This is server streaming no?

Ah crap you're right. I had this backwards the whole time.